docs/config/README.md

```diff
@@ -2561,7 +2561,7 @@ If you have an interesting example not covered there, feel free to share it ther
 | Option        | Default                       | Description                                                                                                                |
 | ------------- | ------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
 | `command`     |                                 | The command whose output should be printed. The command will be passed on stdin to the shell.                              |
-| `when`        |                                 | A shell command used as a condition to show the module. The module will be shown if the command returns a `0` status code. |
+| `when`        | `false`                         | Either a boolean to permanently enable or disable the module or a string shell command used as a condition to show the module. The module will be shown if the command returns a `0` status code. |
```

Comment:
At least for me just "boolean" makes it not completed clear if case matter or not (true, True, TRUE).

Reply:
It refers to TOML booleans. They aren't really explained for `disabled`/`enabled` either but I suppose it might make sense to mention it here.

Reply:
Fun, so `"true"` would still refer to the `true` command?

Reply:
Yes.